GoogleForCreators/web-stories-wp
# Added disabled prop to inputs used in the animation panel

## Context


Alert the user that animations on the first page of stories will not be seen/used because of AMP restrictions and disallow add/edit of animations on the first page.

## Summary


Adds a `disabled` prop and logic to input components within the animations panel, as well as a warning indicating why these inputs are disabled

## Relevant Technical Choices


Per the [AMP spec](https://amp-experiments-dashboard.vercel.app/) `story-disable-animations-first-page` - animations are not supported on the first page of stories.

## To-do



## User-facing changes


before/after
https://user-images.githubusercontent.com/14057928/166295624-9ec0fb08-c782-4f36-91e9-b886ac5ff475.mp4

## Testing Instructions






This PR can be tested by following these steps:

1. Go to Story Editor
2. Add components to the 'canvas' that can have animations (Shapes, Images, Background, etc)
3. Try to add/change the animations of those components

On Staging:
- Notice you are able to add and change animations on all pages of the story

On Branch:
- Notice you are not able to add or change animations on the first page of the story, you should be able to add/change animations on all other pages.
- Notice a waring dialog on the animation panel for the first page
- 4. Moving the pages to and from the first page slot should update the animation panel accordingly


## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #11321
Fixes #11322
